_config/products.yml

```diff
@@ -478,11 +505,33 @@ jbt_core:
   url_path_fragment: jbosstools
   streams:
     mars:
-      4.3.0.Alpha2:
-        update_site_url: http://download.jboss.org/jbosstools/mars/development/updates/core/
-        # No marketplace before Beta marketplace_url: http://marketplace.eclipse.org/node/1617241 
-        # No marketplace before Beta marketplace_install_url: http://marketplace.eclipse.org/marketplace-client-intro?mpc_install=1617241
+      4.3.0.Beta1:
+        update_site_url: http://download.jboss.org/jbosstools/mars/development/
```

Comment:
the url used in marketplace is http://download.jboss.org/jbosstools/mars/development/updates/ 
I'll use that instead.